README.md

```diff
@@ -7,7 +7,7 @@ Provided through [Tailwind CSS IntelliSense](https://github.com/tailwindlabs/tai
 ### Installation
 
 * Install [LSP](https://packagecontrol.io/packages/LSP) and `LSP-tailwindcss` via Package Control.
-* In order for the extension to activate you must have [tailwindcss](https://tailwindcss.com/docs/installation) installed and a [Tailwind config file](https://tailwindcss.com/docs/installation#create-your-configuration-file) named `tailwind.config.js` in your workspace.
+* In order for the extension to activate you must have a [Tailwind config file](https://tailwindcss.com/docs/installation#create-your-configuration-file) named `tailwind.config.js` in your workspace.
```

Comment:
- In ST ecosystem there are no "extensions".
- Not sure if "activate" is the right word. The server will start anyway, it will just not function (or not function properly at least).

So it could be rewritten to something like "In order for the package to function properly you must have a ...."

Reply:
is `"In order for LSP-tailwindcss to work, you must have a..."` better?
(if not I will use your suggestion)